### PR TITLE
Make Contract Addresses configurable

### PR DESCRIPTION
**Pull request by mabalashov**
Updated Contract Addresses
Implemented the method `setContractAddresses` to be able to change the Contract Addresses
Renamed `constants.ts` -> `config.ts` due to it is not constants anymore
Renamed `contracts.d.ts` -> `contracts.ts` due to was not able to import types

### CODE REVIEW

File: src/contracts/config.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  bet: {
+    get address() {
+      return state.contractAddresses.bet || '0x4F0Dc3aAD27379E78C0777f66a07c2ba61B66C71'
+    },
+    abi: AzuroBetABI,
+    decimals: 18,
+  },
+  usdt: {
+    get address() {
+      return state.contractAddresses.usdt || '0xf5f125ffFFe359f2Bfe44776B5604eDFa82A0Ff2'
```
Comment (pavelivanov) [inline src/contracts/config.ts:32]: remove default addresses from here.

File: src/contracts/state.ts [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+  }
+  if (contractAddresses.usdt) {
+    state.contractAddresses.usdt = contractAddresses.usdt;
+  }
+  if (contractAddresses.core) {
+    state.contractAddresses.core = contractAddresses.core;
+  }
+  if (contractAddresses.lp) {
+    state.contractAddresses.lp = contractAddresses.lp;
+  }
```
Comment (pavelivanov) [inline src/contracts/state.ts:33]: remove these conditions. User should pass all addresses w/o exceptions.

Comment (pavelivanov) [reply src/contracts/state.ts:33]: add description to the readme that user should pass addresses during configuration process.

https://sdk-docs-eta.vercel.app/get-started#configure

Repo: https://github.com/Azuro-protocol/sdk-docs

**Review by pavelivanov — CHANGES_REQUESTED** [review]

**Comment by mabalashov** [comment]
BTW, do you keep the commit history clear?
I see several commits were created during the merging. Actually, I am not sure, if it possible to squash commits of PR during the merging it in the main branch in GitHub
So, please let me know, if it is important

**Comment by mabalashov** [comment]
https://github.com/Azuro-protocol/sdk/pull/2

**Pull request closed without merging**